packages/logging_cloudwatch/aws_logging_cloudwatch/lib/src/queued_item_store/queued_item_store.dart

```diff
@@ -38,6 +38,9 @@ class QueuedItem {
     required this.timestamp,
   });
 
+  /// Getter for byte size of the item.
```

Comment:
```suggestion
  /// The size of the item, in bytes, when stored in the database.
```

Mentioning it's a getter doesn't add much value. Prefer using more descriptive comments: https://dart.dev/effective-dart/documentation#prefer-starting-a-non-boolean-variable-or-property-comment-with-a-noun-phrase